packages/node-core/src/db/sync-helper.ts

```diff
@@ -276,6 +276,17 @@ export function getExistedIndexesQuery(schema: string): string {
   return `SELECT indexname FROM pg_indexes WHERE schemaname = '${schema}'`;
 }
 
+export async function getDbSizeQuery(sequelize: Sequelize, schema: string): Promise<number> {
```

Comment:
I think we can have a query that does everything in one and updates the metadata table if its not been update in a certain time